test/opsem/sext.01.ll

```diff
@@ -0,0 +1,46 @@
+; RUN: %seabmc "%s" 2>&1 | %oc %s
+; RUN: %seabmc --horn-bv2-lambdas --log=opsem3 "%s" 2>&1 | %oc %s
+
+; CHECK: ^sat$
+; ModuleID = 'sext.01.ll'
+;; tests the behaviour of sext (sign extension)
+source_filename = "../test/bmc/test-bmc-1.false.c"
+target datalayout = "e-m:o-p:32:32-f64:32:64-f80:128-n8:16:32-S128"
+target triple = "i386-apple-macosx10.13.0"
+
+declare i32 @nd() local_unnamed_addr #0
+
+declare void @verifier.assume(i1)
+declare void @verifier.assume.not(i1)
+declare void @seahorn.fail()
+
+; Function Attrs: noreturn
+declare void @verifier.error() #1
+
+; Function Attrs: nounwind ssp uwtable
+define i32 @main() local_unnamed_addr #2 {
+entry:
+  %nd1 = call i32 @nd()
+  %z = sext i1 false to i32
+  %sum = add i32 %nd1, %z
+  %e = icmp eq i32 %nd1, %sum
+  call void @verifier.assume(i1 %e)
+  br label %verifier.error
```

Comment:
this is a good example on how a test should look like.

Make sure there are both `sat` and `unsat` tests for this.